src/plugins/intel_gpu/tests/unit/test_cases/permute_gpu_test.cpp

```diff
@@ -75,6 +92,52 @@ TEST(permute_gpu_f32, output_ordering_test)
     }
 }
 
+TEST(permute_f_and_y_gpu_f32, output_ordering_test)
+{
+    auto& engine = get_test_engine();
+
+    std::vector<std::vector<int32_t>> input_tensors = {
+        { 1, 256, 1024, 1 },
+    };
+    std::vector<std::vector<uint16_t>> permutations = {
+        { 0, 2, 1, 3 }, //do nothing
```

Comment:
what does this "do  nothing" comment?
Also this test looks like just checking the shape... is it meaningful test?

Reply:
Reverted.